### PR TITLE
Update brigade-information for last weeks changes

### PR DESCRIPTION
**Pull request by tdooner**

### CODE REVIEW

File: organizations.json [hunk 1]
```diff
@@ -1401,7 +1401,7 @@
         ]
     },
     {
-        "name": "Code for Raleigh",
+        "name": "Open Raleigh Brigade",
```
Comment (tdooner) [inline organizations.json:1406]: Hey @jhibbets, can you confirm that the word "Brigade" is officially part of the name - and should display in all places next to "Open Raleigh"?

Comment (tdooner) [reply organizations.json:1406]: Did any of the other web properties change as a result of the rename?

**Comment by jhibbets** [comment]
It should be listed as "Open Raleigh Brigade" because Open Raleigh is the
open data program from the City. The new website will be raleighbrigade.com,
however we don't have anything there at the moment, but are working on it
this week.

**Comment by tdooner** [comment]
@jhibbets Great, thanks. Let me know when you launch the new website and I'll update this, or feel free to make the PR yourself.

**Pull request merged**